src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -594,6 +504,28 @@ void Graph::InitEdges() {
         numberOfEdges--;
     };
 
+    // In case of edge's child is 'Convert' op and the edge's input/output has different precision, if set convert's input
+    // precision same with edge parent precision, it will avoid additional new convert op to be added.
+    for (ptrdiff_t i = 0; i < numberOfEdges; i++) {
```

Comment:
yes, it will be replaced by PR19651 and/or another transformation PR.

Reply:
It seems Reorder node improvement ([PR19651](https://github.com/openvinotoolkit/openvino/pull/19651)) still cannot solve "unsupported fp 64 reorder" problem, the same failure cases:

![image](https://github.com/openvinotoolkit/openvino/assets/31196718/046618a7-82f9-44f8-bd4e-e6b6ad23d613)

Because dnnl::reorder doesn't support fp64, but the test graph need insert a fp64 reorder in these test cases.
We should solve this problem in graph level rather than previous suggested node level.
One solution is [PR19332](https://github.com/openvinotoolkit/openvino/pull/19322)
Another solution is to investigate transformation_pipeline to solve it.

Reply:
Reorder PR (https://github.com/openvinotoolkit/openvino/pull/19651) can solve all unsupported precision reoder issue, but there are several unsupported precision issue for import/export model tests( I16/U16/U32/U64)